app/scripts/modules/netflix/ci/services/build.read.service.spec.ts

```diff
@@ -0,0 +1,136 @@
+import * as moment from 'moment';
+import {IHttpBackendService, IPromise, mock} from 'angular';
+
+import {SETTINGS} from 'core/config/settings';
+import {Api, API_SERVICE} from 'core/api/api.service';
+import {
+  CI_BUILD_READ_SERVICE, CiBuildReader, ICiBuild,
+  ICiBuildOutputConfig
+} from 'netflix/ci/services/build.read.service';
+import {CI_FILTER_MODEL, CiFilterModel} from 'netflix/ci/ci.filter.model';
+
+describe('CiBuildReader', () => {
+
+  const CI_BUILD_URL = `${SETTINGS.gateUrl}/ci/builds`;
+  let $http: IHttpBackendService;
+  let API: Api;
+  let ciBuildReader: CiBuildReader;
+  let ciFilterModel: CiFilterModel;
+
+  beforeEach(mock.module(API_SERVICE, CI_BUILD_READ_SERVICE, CI_FILTER_MODEL));
+  beforeEach(mock.inject((_$httpBackend_: IHttpBackendService,
+                          _API_: Api,
+                          _ciBuildReader_: CiBuildReader,
+                          _CiFilterModel_: CiFilterModel) => {
+    $http = _$httpBackend_;
+    API = _API_;
+    ciBuildReader = _ciBuildReader_;
+    ciFilterModel = _CiFilterModel_;
+  }));
+
+  beforeEach(() => {
+    ciFilterModel.searchFilter = 'none';
+  });
+
+  afterEach(() => {
+    ciFilterModel.searchFilter = null;
+  });
+
+  afterEach(function () {
+    $http.verifyNoOutstandingExpectation();
+    $http.verifyNoOutstandingRequest();
+  });
```

Comment:
You have two `afterEach` calls, and only one uses an arrow function. Why not collapse them into a single `afterEach`?